crawl-ref/source/mon-clone.h

```diff
@@ -1,12 +1,15 @@
 #pragma once
 
+#include "mon-attitude-type.h"
```

Comment:
Hm, it's unusual for us to include a header from another header. I see why you did it here: it seems necessary to have ATT_SAME in scope. Maybe @neilmoore has a suggestion of another way to do this.